doc/example/reference.cpp

```diff
@@ -379,10 +373,11 @@ MOCK_CLASS( mock_class )
 namespace function_example_1
 {
 //[ function_example_1
-MOCK_FUNCTION( f, 1, float( int ) )
+MOCK_FUNCTION( f, 1, void( int ) )
 
 BOOST_AUTO_TEST_CASE( demonstrates_instantiating_a_mock_function )
 {
+    MOCK_EXPECT(f).once().with(3);
```

Comment:
It’s a bit too bad that we can’t keep the introductory docs as minimal as possible, I think.

Reply:
Sure it is a little bit more but at least for me it is now easier to understand as you got some more context. And it actually tests what it says it would do

Reply:
Right, that one is probably fine.